view/adminhtml/web/js/menu.js

```diff
@@ -1,12 +0,0 @@
-require([
```

Comment:
@thomas-kl1 Not sure if this file is knowingly removed. This code was introduced in recent release to have a small non interrupting "powered by" link.

There is already a feedback/issue for this https://github.com/redchamps/clean-admin-menu/issues/19 but we are still evaluating if we should keep it or revert the change.

Reply:
Yes I removed the file on purpose. I don't make sense ta have such link regarding the purpose of the module.

Reply:
That's vszdnbecausebat the end the Magento looks like a Christmas tree. Adding new menu entries does not affect usability big merchants then.
I think adding code that does not have any value for merchant is a loss of quality, performance and is not eco-friendly

Reply:
Absolutely, it has no benefit for the merchant & it should be removed from the merchant perspective. But from our perspective, it serves a value to us & doesn't impact merchant much logically.

But please allows us some time to figure this out, probably we will go ahead to remove it or look for some alternative approach.